corehq/messaging/scheduling/static/scheduling/js/dashboard.js

```diff
@@ -1,6 +1,10 @@
-/* globals d3, nv, ko */
-hqDefine("scheduling/js/dashboard", function () {
-    var dashboardUrl = hqImport("hqwebapp/js/initial_page_data").reverse("messaging_dashboard");
+hqDefine("scheduling/js/dashboard",[
+    'jquery',
+    'knockout',
+    'hqwebapp/js/initial_page_data',
+    'nvd3/nv.d3.min',
+], function ($, ko, initialPageData) {
```

Comment:
Can you add `d3` and `nv` to the parameter list (and d3 to the dependency list)? It'll work either way, since d3 and nvd3 expose globals, but for best practices' sake, let's treat them like modules rather than globals.